docker/helm/templates/kyuubi-service.yaml

```diff
@@ -18,13 +18,18 @@
 apiVersion: v1
 kind: Service
 metadata:
-  name: kyuubi-nodeport
+  name: kyuubi-svc
+  labels:
+    app: {{ template "kyuubi.name" . }}
+  annotations:
+  {{ toYaml .Values.service.annotations | indent 4 }}
```

Comment:
nit: two more indention, just for readable

Reply:
Done.